app/frontend/src/features/messages/requests/HostRequestSendField.tsx

```diff
@@ -131,7 +139,11 @@ export default function HostRequestSendField({
               HostRequestStatus.HOST_REQUEST_STATUS_PENDING ||
               hostRequest.status ===
                 HostRequestStatus.HOST_REQUEST_STATUS_REJECTED) && (
-              <FieldButton callback={handleAccept} isLoading={isButtonLoading}>
+              <FieldButton
+                callback={handleAccept}
+                isLoading={isButtonLoading}
+                isSubmit={ButtonTypes.button}
```

Comment:
Once `isSubmit` is a boolean, you can just do `isSubmit` for true or omit it for false/undefined. Same for the other cases.